src/utils/ledger.js

```diff
@@ -125,6 +157,61 @@ if (IPC_RENDERER) {
       }
       IPC_RENDERER.send("ledger:getSignatures", arr);
     },
+
+    /**
+     * Sign token info on ledger
+     *
+     * @param {Object} token
+     *  with uid (hex), symbol and name
+     *
+     * @memberof Ledger
+     * @inner
+     */
+    signToken(token) {
+      const data = Buffer.concat(serializeTokenInfo(token, false));
+
+      IPC_RENDERER.send("ledger:signToken", data);
+    },
+
+    /**
+     * Send tokens to ledger to create context for sign_tx
+     *
+     * @param {Object} tokens
+     *  each with uid (hex), symbol, name and signature
+     *
+     * @memberof Ledger
+     * @inner
+     */
+    sendTokens(tokens) {
+      const data = tokens.map(t => Buffer.concat(serializeTokenInfo(t, true)));
```

Comment:
Should we add a validation here on the number of tokens to avoid hitting the `TX_MAX_TOKENS` in the Ledger App?

Reply:
This validation is done on another PR that calls this method, actually the user won't be able to add more than 10 tokens to the transaction if it's a hardware wallet.

Reply:
Same thing I commented in the other comment.

Maybe we want to make sure there is a validation here as well, in case someone uses this method somewhere else besides your component.